homeassistant/components/climate/const.py

```diff
@@ -45,6 +45,9 @@
 # Device is in comfort mode
 PRESET_COMFORT = 'comfort'
 
+# Device is in a user-configurable mode
+PRESET_CUSTOM = 'custom'
```

Comment:
Let's not add this to the official constants. The official constants are there to standardize some things to help with translations/icons/Google/Alexa. Custom does not fit that bill and you can keep it inside evohome. There is no limitation on what values you return from the `preset_modes` property.

Reply:
OK, will do.  

I previously added `PRESET_ACTIVITY` for **geniushub** - it has movement sensors - is that OK?

Reply:
OK, thanks.

Reply:
yes, because that can be used by others